frontend/services/entities/host_count.ts

```diff
@@ -18,7 +17,7 @@ export interface IHostCountLoadOptions {
   perPage?: number;
   selectedLabels?: string[];
   globalFilter?: string;
-  status?: string;
+  status?: "online" | "offline" | "new";
```

Comment:
Only allows these strings

Reply:
nice! since the same three strings are used in multiple places, should we consider adding a type?

```ts
type HostStatus = "online" | "offline" | "new"

export interface ISortOption {
  // ...
  status?: HostStatus,
}
```